editor/utils/test/terms.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Internal dependencies
+ */
+import { buildTermsTree } from '../terms';
+
+describe( 'buildTermsTree()', () => {
+	describe( 'isHorizontalEdge', () => {
```

Comment:
Some copy-paste that went wrong? 😄

Reply:
Yes @mcsf  I missed it, it was solved :D